packages/cli/src/codemods/transforms/transform-grid-options-v33-0/__fixtures__/scenarios/jsx/transformGroupHideParentOfSingleChild/input.jsx

```diff
@@ -0,0 +1,87 @@
+import { AgGridReact } from '@ag-grid-community/react';
+
+function MyComponent (props) {
+  return (
+    <AgGridReact
+      groupRemoveSingleChildren
+    />
+  )
+}
+
+function MyComponent2 (props) {
+  return (
+    <AgGridReact
+      groupRemoveSingleChildren={true}
+    />
+  )
+}
+
+function MyComponent3 (props) {
+  return (
+    <AgGridReact
+      groupRemoveSingleChildren={false}
+    />
+  )
+}
+
+function MyComponent4 (props) {
+  const [groupRemoveSingleChildren, setGroupRemoveSingleChildren] = useState(true);
+  return (
+    <AgGridReact
+      groupRemoveSingleChildren={groupRemoveSingleChildren}
+    />
+  )
+}
+
+function MyComponent5 (props) {
+  return (
+    <AgGridReact
+      groupRemoveLowestSingleChildren
+    />
+  )
+}
+
+function MyComponent6 (props) {
+  return (
+    <AgGridReact
+      groupRemoveLowestSingleChildren={true}
+    />
+  )
+}
+
+function MyComponent7 (props) {
+  return (
+    <AgGridReact
+      groupRemoveLowestSingleChildren={false}
+    />
+  )
+}
+
+function MyComponent8 (props) {
+  const [groupRemoveLowestSingleChildren, setGroupRemoveLowestSingleChildren] = useState(true);
+  return (
+    <AgGridReact
+      groupRemoveLowestSingleChildren={groupRemoveLowestSingleChildren}
+    />
+  )
+}
+
+function MyComponent9 (props) {
+  const [groupRemoveLowestSingleChildren, setGroupRemoveLowestSingleChildren] = useState(true);
+  return (
+    <AgGridReact
+      groupRemoveSingleChildren
+      groupRemoveLowestSingleChildren={groupRemoveLowestSingleChildren}
+    />
+  )
+}
+
+function MyComponent10 (props) {
+  const [groupRemoveLowestSingleChildren, setGroupRemoveLowestSingleChildren] = useState(true);
+  return (
```

Comment:
Are we doing anything for Angular and Vue?

Reply:
No - but historically we haven't anyway, it seems Tims stuff didn't work for angular (correct me if I'm wrong?)

Reply:
Might just be a case of checking if the JS version tweaks Angular code that is defined in the component, in a way that then does not match the template. Same potentially for Vue, the code might be updated by the JS one too.